doc/dune-files.rst

```diff
@@ -1,8 +1,8 @@
 ****************
-Stanza reference
+Stanza Reference
 ****************
 
-dune-project
+Dune-Project
```

Comment:
Should be `dune-project` since that is the name of the file Dune expects.

Reply:
Got it! Will you reject this change or shall I change it on my file and push again?

Reply:
The latter sounds like the way to go. The other changes in this pull request are correct.

Reply:
Great! I’ll do that today.